src/main/java/com/volunteer/enterprise/VolunteerAppController.java

```diff
@@ -15,18 +15,14 @@
  VolunteerService volunteerService;
 
 
-    @RequestMapping("/")
-    public String index(Model model){
-
-Volunteer volunteer = new Volunteer();
-
-volunteer.setVolunteerID(1);
-volunteer.setVolunteerUser("Iron Man");
-volunteer.setVolunteerPass("1234");
-
-model.addAttribute(volunteer);
-        return "Index";
-
+ @RequestMapping("/")
+ public String index(Model model){
+     Volunteer volunteer = new Volunteer();
+     volunteer.setVolunteerID(1);
+     volunteer.setVolunteerUser("Iron Man");
+     volunteer.setVolunteerPass("1234");
+     model.addAttribute(volunteer);
+     return "Index";
     }
```

Comment:
Control Alt I will do this automatically in IDEA.  Also, include the last curly, which has indents off.